src/Pages/Volunteer.jsx

```diff
@@ -448,6 +463,156 @@ export const Volunteer = () => {
                   writing.
                 </FormHelperText>
               </FormGroup>
+              <Typography
+                variant="body1"
+                sx={{ fontWeight: "bold", pt: 2 }}
+              >
+                Volunteer Hold Harmless Agreement:
+              </Typography>
+              <FormHelperText>
+                Agreement made this {currentDateAsOrdinal} day of {currentMonth}, {currentYear} by and between A Refuge for Saving the Wildlife, Incorporated&copy; (hereinafter referred to as RSW&copy;), located in Northbrook, IL, and (hereinafter referred to as Volunteer).
```

Comment:
I know this was how it was originally written, but the last "and" kinda leaves me hanging. 😆 Maybe we should try something like "and {firstname} {lastname} (hereinafter referred to as Volunteer)"?

Reply:
That's a good point! Since we only have a "Name" field currently, I can just add the user's full name into the text here. Maybe later we can talk about whether we want to have separate fields for first name and last name? I could try to write a function that splits and formats the first and last names from the "Name" field, but I feel like there are too many uknowns with how people might enter their names. What do you think?

Reply:
I forgot we only have one name field! I think it's fine to just use the field we have. 👍